pages/auth2.html

```diff
@@ -144,6 +178,7 @@ <h1>FluffyScratch Auth</h1>
       </div>
     </footer>
     <script>
+      let nextURL = {{nextURL}};
```

Comment:
don't you need to do `"{{nextURL}}"`?

Reply:
nextURL is stringified
https://github.com/hamptonmoore/FluffyScratch/blob/30da43dee1ec6d8fb881efd118624d3306b75aa9/routes/auth/getKeys/v2.mjs#L36

Reply:
If it wasn't, you could just set the redirect url to `example.com";alert("lol");nextURL="https://example.com` and cause XSS